### ISSUE TITLE
Cursor is not changed in webview2

### ISSUE THREAD

**Issue by khoabui1412** [microsoft/microsoft-ui-xaml#8196]
### Describe the bug

Hi,
I have a custom web html:
```
<!DOCTYPE html>
<html>
<head>
	<title>Change cursor on hover</title>
	<style>
		.square {
			width: 100px;
			height: 100px;
			background-color: blue;
		}
		.square:hover {
			cursor: crosshair;
		}
	</style>
</head>
<body>
	<div class="square"></div>
</body>
</html>
```

But when open it by webview2 in winUI3, the cursor is not changed to crosshair as expected.

### Steps to reproduce the bug

Open browser has custom cursor by webview2

### NuGet package version

Windows App SDK 1.2.3: 1.2.230118.102

### Packaging type

Unpackaged

### Windows version

Windows 11 version 22H2 (22621, 2022 Update), Windows 11 version 21H2 (22000)

### IDE

Visual Studio 2022

**Comment by DmitriyKomin** [comment]
Thanks for reporting this @khoabui1412, indeed the HTML cursor types _crosshair_, _help_, _move_ and _not_allowed_ were not getting mapped correctly. This is now fixed internally (tracked by ADO Bug 45873654), expected to be included in WinAppSDK 1.5.

**Comment by khoabui1412** [comment]
@DmitriyKomin, does that fix include custom (external) image for cursor?

**Comment by DmitriyKomin** [comment]
No, it does not. That is a separate issue, tracked by MicrosoftEdge/WebView2Feedback#1925. @champnic on Edge team may have more info.

**Comment by bpulliam** [comment]
Removing the duplicate tag, marking as Fixed, and adding to 1.5 Milestone since the original issue of HTML cursor types is fixed in 1.5.
